paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -32,22 +29,21 @@ import org.gradle.api.tasks.TaskAction
 
 @CacheableTask
 abstract class PrepareResourcesTask : DefaultTask() {
+
   @get:Input
   abstract val packageName: Property<String>
 
-  @get:InputDirectory
-  @get:PathSensitive(PathSensitivity.RELATIVE)
-  abstract val mergeResourcesOutput: DirectoryProperty
+  @get:Input
+  abstract val mergeResourcesOutput: Property<String>
```

Comment:
It has to be a String? Gradle is so annoying

Reply:
Yep, basically `DirectoryProperty` with `@PathSensitive(PathSensitivity.RELATIVE)` is used for a directory where you want an sub folder or file changes to run the task. Since its relative and the base directory is `.../debug` and `.../release` gradle sees the underlying files as the same.

The suggestion for using Property<String> here is if you only want the path to be used as an input and the contents don't matter. This was our exact usecase as we just use the path in the generated `resources.txt` file.

Reply:
The worst is when Gradle simply ignores your parameter without telling you it's not the right type and that what you expect to happen is never gonna be!